src/hooks/useInitializeWallet.js

```diff
@@ -66,13 +66,18 @@ export default function useInitializeWallet() {
         // Load the network first
         await dispatch(settingsLoadNetwork());
 
+        // When the `seedPhrase` is not defined in the args, then
+        // this means it's a new fresh wallet created by the user.
+        const defaultIsNew = typeof seedPhrase === 'undefined';
```

Comment:
since the `seedPhrase` is passed along to the `walletInit` argument and that's all that is used to determine `defaultIsNew`, we could do the check inside of `walletInit` without needing to pass along `defaultIsNew` as an additional argument